tests/test_session.py

```diff
@@ -538,7 +538,7 @@ def test_get_set_state_on_solver(new_solver_session):
 
 def test_solver_structure(new_solver_session):
     solver = new_solver_session
-    with pytest.warns(PyFluentDeprecationWarning):
+    with pytest.warns(DeprecationWarning):
```

Comment:
Was this change intentional? In deprecation.py it is still raising a "PyFluentDeprecationWarning"

Reply:
Yes, we are using @deprecated from deprecated.sphinx so we don't need to raise any custom warning manually. It raises on it's own as shown in the PR description.